ERCS/erc-____.md

```diff
@@ -0,0 +1,492 @@
+---
+eip: ____
+title: "Cross-chain Storage Router Protocol"
```

Comment:
```suggestion
title: Cross-chain Storage Router Protocol
```
Quotes are not required

Reply:
Fixed in https://github.com/ethereum/ERCs/pull/404/commits/7af50f32660600c2321aa5ba29af24322bd8c304